script/Ciel/Day 9/09_01_CIEL09_9.txt

```diff
@@ -499,44 +511,44 @@ I want to arrive at that conclusion.
 [sha:3e7d76cc9e783db5a9b31dd5446926d44fbf36df]{
 -- Page 794, Offset 24239.
 -- その状態こそ、何も加わらず何も失われない不変のもの。
-That is the state which is truly one of permanence;
- onto which nothing can be added,
+That is the state of true permanence;
+ one onto which nothing can be added,
  and from which nothing can be taken away.
```

Comment:
You don't need the semi-colon in this case. It should be a period.
Though the triple which is gone either way...